docs/mpdev-references.md

```diff
@@ -113,3 +113,66 @@ properties:
 ```
 
 Properties in `/data-test/schema.yaml` will overlay properties in `/data/schema.yaml`. This can also be used overwrite existing default values for verification.
+
+## Installing a published Marketplace app
+
+`mpdev` is intended for developing your application prior to publishing to
+GCP Marketplace. It can also be used to deploy a published Marketplace app
+following these steps.
+
+### Locating the desired deployer image
+
+From an app listing details page on Marketplace (for example
+[nginx](https://console.cloud.google.com/marketplace/details/google/nginx)),
+locate the container repository path on the left hand side. Also notice
+the image tag under `Container images` section.
+
+![Repository path on listing details page](images/repository-path.png)
+
+The desired deployer image is `[REPOSITORY_PATH]/deployer:[TAG]`.
+Pull and make the image available locally.
+
+```shell
+IMAGE=[REPOSITORY_PATH]/deployer:[TAG]
+
+docker pull $IMAGE
+```
+
+Inspect the schema of the deployer to determine the following:
+- Schema version
+- Parameters to fill out
+
+```shell
+docker run --rm --entrypoint /bin/bash $IMAGE -c 'cat /data/schema.yaml'
+```
+
+### Getting a copy of deploy parameters
+
+To deploy an application, its required parameters have to be filled out.
+One way is to fill out the parameters yourself. Another way to obtain these
+parameters is to create a deployment using Marketplace UI and make a copy
+of the parameters generated by the UI form.
+
+The following works for a schema v2.
```

Comment:
The following works for a schema v2 assuming you are following the instructions for copying the parameters from an already deployed image.